install-dependencies.sh

```diff
@@ -62,3 +64,31 @@ esac
 # needed for unit tests
 sysctl -w fs.aio-max-nr=10485760 || true
 curl -1sLf "https://raw.githubusercontent.com/vectorizedio/seastar/master/install-dependencies.sh" | bash
+
+set -e
+
+root="$(cd $(dirname ${BASH_SOURCE[0]}) && pwd)"
+
+cd $root
+
+rm -rf $root/depot_tools
+git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
+cd $root/depot_tools
+git checkout ecdc362593cfee1ade115ead7be6c3e96b2e7384
+mkdir $root/depot_tools/installed-gn
+
+platform_for_gn=""
+
+case $(uname -i) in
+  x86_64)
+    platform_for_gn="amd64"
+    ;;
+  *)
+    platform_for_gn="arm64"
+    ;;
+esac
+
+curl -L "https://chrome-infra-packages.appspot.com/dl/gn/gn/linux-${platform_for_gn}/+/COENCtFXQmybPdz3KQBjmuSaCK4qdzmzwuCJHQKxovEC" --output $root/depot_tools/gn_zip
+unzip -d $root/depot_tools/installed-gn/ $root/depot_tools/gn_zip
```

Comment:
is this something we should do in oss.cmake as part of the download/setup steps that need to be completed before building v8 as a dependency?

Reply:
For vtools build we use task (not 3rdparty.cmake) for downloading gclient and gn. I thought that use install-dependencies was ok for open-source build. https://github.com/vectorizedio/vtools/pull/173#discussion_r641655810

I can move it.